src/main/kotlin/com/exactpro/th2/common/Module.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020-2022 Exactpro (Exactpro Systems Limited)
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.common
+
+/**
+ * Marker interface for module
+ */
+interface Module
```

Comment:
Let's use this interface as limitation for module classes. I mean generic class and method, for example
`fun <M : Module> method(clazz: Class<M>): M`